src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -123,9 +138,11 @@ protected override void BeginInvokeJS(long asyncHandle, string identifier, strin
             }
         }
 
-        Log.BeginInvokeJS(_logger, asyncHandle, identifier);
+        var invocationInfoJson = invocationInfo.ToJson();
```

Comment:
We shouldn't serialize things to a big string here.

Reply:
Yes, those are not part of the public API, so it's fine, we can change those whenever we want. We can't avoid serializing the arguments as a string, but we don't want to wrap that string into something else and serialize that too.